README.md

```diff
@@ -1,41 +1,8 @@
-<a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
+<a id="readme-top"></a>
```

Comment:
_note: Since there is no open PR in the fronted project for me to review, I'll put my comments here.  Review based on the `dev` branch._

_note 2: Due to the Rails migrations issue, I cannot review the project's functionality in the browser.  Kindly fix the migrations issue.  The next reviewer will check your project's functionality._

# Frontend review comments (documentation and code only):

- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

- [ ] This note:

![image](https://user-images.githubusercontent.com/57421823/224118579-1b646772-d4fd-4118-aecb-088a0f8625c0.png)

This note is just a placeholder intended to give you tips on how to add a license.  Kindly remove it after following its instructions.

- [ ] This section has not been filled up:

![image](https://user-images.githubusercontent.com/57421823/224118762-19b268f7-bf8b-4b7b-a77d-e968a7eb4ed0.png)

Kindly either fill it with a faq, or completely remove it since it's optional. 👍 

- [ ] Please, kindly refactor the App component to ES6 arrow function, so your code complies with the latest standards.


Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

Reply:
Thanks for the review, the frontend consume the already deployed api, if you try to run the frontend on your local machine it will work without running the rails on your local machine. Thanks once again I really appreciate your feedback